daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -285,6 +428,51 @@ func (d *Driver) Create(id, parent, mountLabel string, storageOpt map[string]str
 	return label.Relabel(path.Join(subvolumes, id), mountLabel, false)
 }
 
+// Parse btrfs storage options
+func (d *Driver) parseStorageOpt(storageOpt map[string]string, driver *Driver) error {
+	// Read size to change the subvolume disk quota per container
+	for key, val := range storageOpt {
+		key := strings.ToLower(key)
+		switch key {
+		case "size":
```

Comment:
But I think it needs to be updated in the documentation (I noticed the current docs don't mention for which graphdrivers it's supported, which we may want to add)